### PR TITLE
added RN for 10.18.0.127-core

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-18-0/platform-10-18-0.md [hunk 1]
```diff
@@ -243,6 +243,30 @@ Build version</th>
...
+<td>Administration</td>
+<td>Fixed an issue where the <b>Clear</b> button on the <b>Usage statistics</b> page failed to remove applied filters.</td>
+<td>MTM-50302</td>
+<td>10.18.0.93</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>The button name for confirming the revocation of all tokens has been revised to <b>Log out all users and invalidate tokens</b>.</td>
```
Comment (rssag) [inline content/release-10-18-0/platform-10-18-0.md:264]: what was it before? is it relevant?

Comment (BeateRixen) [reply content/release-10-18-0/platform-10-18-0.md:264]: You mean the entire entry? I was also wondering if it is relevant.

Comment (rssag) [reply content/release-10-18-0/platform-10-18-0.md:264]: i mean, is it relevant (for the release notes entry) what it was before?

Comment (BeateRixen) [reply content/release-10-18-0/platform-10-18-0.md:264]: No, not really, it was quite similar (I think it was "Invalidate tokens") so the context should be clear.

**Pull request merged**